ykman/openpgp.py

```diff
@@ -107,6 +107,18 @@ def __str__(self):
             return "Cached (fixed)"
 
 
+@unique
+class POLICY(IntEnum):  # noqa: N801
```

Comment:
I think this should be renamed to `PIN_POLICY` to clarify. The argument to `set-pin-policy` should still be named `policy`.